src/client/resources/web/index.html

```diff
@@ -1,26 +1,30 @@
-<!DOCTYPE html>

```

Comment:
Also here?

Reply:
Also editorconfig, probably the line endings.